### DISCUSSION TITLE
ZWave-JS is unable to connect to AEON Labs Z‐Stick Gen5 USB Controller stick after add-on update

### DISCUSSION THREAD

**Discussion by loomyr** [zwave-js/zwave-js#6463] [Request Support / Investigate Issue] [answered]
### Checklist

- [X] I have read and followed the above instructions
- [X] I have checked the [troubleshooting section](https://zwave-js.github.io/node-zwave-js/#/troubleshooting/index) and my problem is **not** described there.

- [X] I have read the [changelog](https://github.com/zwave-js/node-zwave-js/blob/master/CHANGELOG.md) and my problem was **not** mentioned there or the fix **did not work**.

### Describe the issue

Since september 2023, I encountered an issue with the Zwave-js UI addon:

- After an update of the Zwave-js UI addon is performed, I receive the following error message:

```
Driver: Failed to open the serial port: Error: No such file or directory, cannot open /dev/ttyACM0 (ZW0100)
Error: Driver: Failed to open the serial port: Error: No such file or directory, cannot open /dev/ttyACM0 (ZW0100)
```

This issue is present on two separate installations, located at two different places. Both installations are based on the same hardware (Lenovo Thinkcentre M910q), with the same version of Proxmox, and the same version of Home Assistant (OS, Supervisore, Core, Frontend).

I have identified two workarounds to temporarily resolve this problem:

> - Shutdown of the Home Assistant virtual machine (VM), followed by disconnecting and reconnecting the USB device in Proxmox. Afterward, I restart the Home Assistant VM. This operation allows Zwave-js to correctly detect my USB key.

or

> - ShutdownHome Assistant VM, followed by a complete restart of the Proxmox hypervisor. After the Proxmox restart, I restart the Home Assistant VM. Once again, this enables Zwave-js to recognize my USB key.

It's important to note that a regular VM reboot doesn't fix the issue. I have to use one of the workarounds mentioned above to recover a working state on my Zwave controller stick.

These temporary workarounds suggest that the issue may be related to how Home Assistant and Zwave-JS interacts with the USB device after the update of the Zwave-js UI addon.

### Software versions

Zwave-js : 
- zwave-js-ui: 9.3.0
- zwave-js: 12.2.3

Current Home Assistant Version:
- Home Assistant 2023.10.5
- Supervisor 2023.10.1
- Operating System 11.1
- Frontend 20231005.0 - latest

Hypervisor where is hosted HomeAssistant VM:
- Proxmox Version: VE 7.4-3

### Device information

Manufacturer: AEON Labs
Model name: Z‐Stick Gen5 USB Controller
Node ID: 001
FW: v1.1
SDK : v6.51.10

### Checklist

- [X] I [made sure](https://zwave-js.github.io/node-zwave-js/#/troubleshooting/index?id=rule-1-use-the-driver-log-luke) to provide a **driver log** on level **debug**.

- [ ] The log includes a re-interview of the problematic device _(if applicable)_.

- [ ] The log includes the problematic interaction with the device _(if applicable)_.

- [ ] I provided the node ID of the problematic device _(if applicable)_.

### Upload Logfile

Logging had not been enabled since the previous issue. I activated logging after the Zwave-JS update and before proceeding with my workaround to restore normal operation.

The logs include then:
1. The state when the add-on was initially started and unable to connect to the USB device **(from: 2023-10-27 17:21:58.956 to: 2023-10-27 17:30:24.208)** 
2. Subsequent logs captured after successfully proceeding with the workaround and returning to normal operation **(starting at: 2023-10-27 17:30:24.209)**

[zwave-js-ui-store.zip](https://github.com/zwave-js/node-zwave-js/files/13190949/zwave-js-ui-store.zip)
[node_1.json](https://github.com/zwave-js/node-zwave-js/files/13190952/node_1.json)

[a comment by an automated account is not shown]

**Comment by AlCalzone** [comment, marked as answer]
See the pinned issue: https://github.com/zwave-js/node-zwave-js/issues/6341

Since you're using a VM you either need to configure it properly (however that works), or disable soft-reset in the addon settings.
